### ISSUE TITLE
activate pcntl extension in CLI

### ISSUE THREAD

**Issue by thislg** [le-phare/le-phare.github.io#125]
Add pcntl extension activation requirement in CLI (for Symfony Messenger)

**Comment by pierreboissinot** [comment]
@thislg il faudrait ajouter une sections "fonctions non désactivées", en commençant par celles de cette extension.

Je me demande si c'est optionnel ou non, il me semble que dès qu'on a du mail on a messenger donc a priori oui vu que dans un projet faros on a forcément du mail.

Qu'en pensez vous ?

**Comment by lucasmirloup** [comment]
Ça fait plusieurs fois que le problème arrive donc je valide :+1: @pierreboissinot

**Comment by thislg** [comment]
Messenger n'est pas une dépendance de Faros et n'est pas obligatoire pour envoyer des emails avec Symfony Mailer, en fait Messenger n'est utilisé que s'il est installé, sinon ça envoie simplement en synchrone. Par contre c'est ce qu'on a tendance à utiliser systématiquement pour faire du traitement asynchrone y compris l'envoi des emails donc c'est cohérent de demander les fonctions nécessaires dans les prérequis.

**Comment by pchabbert** [comment]
Si l'extension n'apporte pas de problème a être activée même si elle n'est pas utilisée je ne vois pas de soucis non plus :+1:

**Comment by pierreboissinot** [comment]
merci pour vos retours , je démarre une PR

**Comment by pierreboissinot** [comment]
Les retours d' @alexandrebouillet 
![image](https://github.com/user-attachments/assets/8d0a9a7c-a263-43b5-9b0a-68e10a1e0ba6)

**Comment by lucasmirloup** [comment]
Hello @alexandrebouillet

On est d'accord que le `disable_functions` dans ton message est celui que tu as demandé à l'hébergeur de mettre ?
Le projet en question utilise-t-il Symfony Messenger ?

À première vue j'aurais tendance à penser que ces fonctions pcntl désactivées sont nécessaires pour Symfony Messenger, mais ce n'est peut-être pas le cas.

**Comment by pierreboissinot** [comment]
> Hello @alexandrebouillet
> 
> On est d'accord que le `disable_functions` dans ton message est celui que tu as demandé à l'hébergeur de mettre ? Le projet en question utilise-t-il Symfony Messenger ?
> 
> À première vue j'aurais tendance à penser que ces fonctions pcntl désactivées sont nécessaires pour Symfony Messenger, mais ce n'est peut-être pas le cas.


> On est d'accord que le `disable_functions` dans ton message est celui que tu as demandé à l'hébergeur de mettre ? 

Non, @alexandrebouillet me corrigera si j'ai mal compris mais il s'agit des func désactivées par l'hébergeur.

>À première vue j'aurais tendance à penser que ces fonctions pcntl désactivées sont nécessaires pour Symfony Messenger, mais ce n'est peut-être pas le cas.

Oui, d'où un warning et non une erreur https://github.com/le-phare/le-phare.github.io/pull/189/files#diff-d469def2c8383934ca736810fa3ffc2c362617570ecf2184f2fc12d15f229fb9R31

PS: @lucasmirloup j'imagine que tu n'as pas lu la [PR](https://github.com/le-phare/le-phare.github.io/pull/189), je t'ai ajouté en reviewer
